src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,33 @@
 import React from 'react';
+import { Comment } from '../../Types/Comment';
```

Comment:
Same here and below. Please fix all cases
![image](https://user-images.githubusercontent.com/96409563/197714496-a8ca01f6-9d92-45ed-b73a-b0ce0e713615.png)